backend/src/ml_space_lambda/authorizer/lambda_function.py

```diff
@@ -266,7 +271,13 @@ def lambda_handler(event, context):
                     except Exception as e:
                         logging.exception(e)
                         logging.info("Access Denied. Encountered error while determining resource access policy.")
-
+            elif requested_resource.startswith("/app_configuration"):
```

Comment:
We'll actually need to have this route open to everyone authenticated or not once the OIDC configuration is manageable post deployment. Doesn't need to change in this PR just calling it out as we build out the actual configuration pieces.